src/main/java/com/hashedin/bitly/MappingException.java

```diff
@@ -0,0 +1,11 @@
+package com.hashedin.bitly;
+
+public class MappingException extends RuntimeException {
+
```

Comment:
We create a custom exception to track the case where a short code cannot be found. Notice we extend RuntimeException instead of Exception - this makes it an unchecked exception.